### PR TITLE
Demonstrate materializations for Comb support in RTLToLLHD.

### PR DESCRIPTION
**Pull request by mikeurbach**

### CODE REVIEW

File: test/Conversion/RTLToLLHD/structure.mlir [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// RUN: circt-opt -convert-rtl-to-llhd -split-input-file -verify-diagnostics %s | FileCheck %s
+// RUN: circt-opt -convert-rtl-to-llhd -simple-canonicalizer -split-input-file -verify-diagnostics %s | FileCheck %s
```
Comment (lattner) [inline test/Conversion/RTLToLLHD/structure.mlir:1]: Please keep testcases limited to a specific subsystem.  Making this run the lowering and canonicalizer makes the test more fragile.

Comment (mikeurbach) [reply test/Conversion/RTLToLLHD/structure.mlir:1]: Sorry, just wanted to show that the old tests pass. I'll pull the canonicalization tests out if we want to go this direction.

Comment (lattner) [reply test/Conversion/RTLToLLHD/structure.mlir:1]: thx!

**Comment by mikeurbach** [comment]
#988 was the full implementation.

**Pull request closed without merging**